.github/actions/build_ci/entrypoint.sh

```diff
@@ -3,13 +3,7 @@
 readonly TARGET="$1"
 
 # Known good version for PR testing
-ZEPHYR_SDK_VERSION=v0.17.0
-ZEPHYR_VERSION=v4.1.0
-
-ZEPHYR_TOOLCHAIN_VARIANT=zephyr
-ZEPHYR_SDK_API_FOLDER=https://api.github.com/repos/zephyrproject-rtos/sdk-ng/releases
-ZEPHYR_SDK_VER_SELECT="tags/$ZEPHYR_SDK_VERSION"
-ZEPHYR_SDK_SETUP_TAR=zephyr-sdk-.*linux-x86_64.tar.xz
+ZEPHYR_VERSION=v4.2.0
```

Comment:
Similar to open-amp, I suggest using the latest Zephyr release (4.3), unless there are strict requirements for 4.2. Are there any?

Reply:
updated to 4.3